### ISSUE TITLE
aarch64 wheel

### ISSUE THREAD

**Issue by baca790** [GUDHI/gudhi-devel#876]
Hi all

Would it be possible to put a manylinux aarch64 wheel for gudhi up on pypi?

Immediate relevant use case:  linux devcontainers running on Mac ARM64 architecture - a fairly standard data science workflow these days. 
The current macos_*_universal wheels will run on the native Mac ARM64 hardware, but not in a linux docker devcontainer running on the same, hence the need for the aarch64 build. 

Extended use case: I'm really keen to try gudhi for edge computing applications, such as on NVIDIA Jetson Orin Nano devices. The same aarch64 builds would work on there.

Thanks and regards
Ben Cassidy

**Comment by mglisse** [comment]
Thanks for the suggestion. We will try to add that in the future (no promise about when it will happen). In the mean time, may I suggest using conda? We have a linux-aarch64 package on conda-forge, and it is likely to be faster than the version on pypi (compiled with more recent software).

**Comment by baca790** [comment]
Understood. I'll see if I can make that work with conda in the meantime. Would you be open to a PR if I find the time to work on the pypi version?
Thanks.

**Comment by VincentRouvreau** [comment]
> Understood. I'll see if I can make that work with conda in the meantime. Would you be open to a PR if I find the time to work on the pypi version? Thanks.

Yes, we are open for contributions. Linux pip packaging is happening in `.github/workflows/pip-packaging-linux.yml` and uses the `gudhi/pip_for_gudhi:latest` docker image.
This image is built from https://github.com/GUDHI/gudhi-deploy/blob/main/Dockerfile_for_pip
Do not hesitate if you have any other questions
